src/raster/r.forcircular/r.forcircular.html

```diff
@@ -0,0 +1,64 @@
+<h2>DESCRIPTION</h2>
+
+Analyze and measure the level of sustainability of the forest-wood supply chain in a circular bioeconomy approach.
+
+<h2>NOTES</h2>
+This module permits to identifies potentially usable forest areas both from a technical and economic point of view. The mandatory data input are a vector file with a series of mandatory fields described below:
+<ul>
+    <li>incr_ha: [float] value of the forest increase [mc * ha * year-1]</li>
+    <li>management: [integer] forest management system (form of government), can take on two values, namely (1) for forest management and (2) for coppice management</li>
+    <li>treatment: [integer] treatment adopted, it can take on two values, namely (1) for the final use cut and (2) for the thinning. Thinning is not foreseen for coppice woods.</li>
+    <li>roughness: [integer] level of terrain accidentality - (0): no accidentality; (1): locally bumpy; (2): partially bumpy; (3): mostly bumpy</li>
+    <li>tree_diam: [integer] average diameter of the stems [cm]. If the data is not available, indicate the generic value 99999</li>
+    <li>tree_vol: [decimal] average volume per plant [m3]. If the data is not available, indicate the generic value 9,999</li>
```

Comment:
Homogenize the way of expressing thousands

Reply:
Mi dispiace ma non posso farlo perché i valori sono valori standard nel framework

Reply:
Then, if the second one is a decimal number, please use `.` instead of comma.

Reply:
Done